README.md

```diff
@@ -1,6 +1,12 @@
 # Beamchmark
+[![Hex.pm](https://img.shields.io/hexpm/v/beamchmark.svg)](https://hex.pm/packages/beamchmark)
+[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/beamchmark)
+[![CircleCI](https://circleci.com/gh/membraneframework/beamchmark.svg?style=svg)](https://circleci.com/gh/membraneframework/beamchmark)
 
-**TODO: Add description**
+Tool for measuring EVM performance.
+
+At the moment, the main interest of Beamchmark is scheduler utilization, reductions and context switches number.
+For more information please refer to API docs.
```

Comment:
I would probably add here or maybe in an API some information compring it to `benchee` and when you should use one and when another, but I will leave it up to you.